tools/gn

```diff
@@ -478,7 +478,7 @@ def parse_args(args):
                       help='Strip debug symbols from the output. This defaults to true and has no effect on iOS.')
   parser.add_argument('--no-stripped', dest='stripped', action='store_false')
 
-  parser.add_argument('--prebuilt-dart-sdk', default=False, action='store_true',
+  parser.add_argument('--prebuilt-dart-sdk', default=True, action='store_true',
```

Comment:
The text of the warning on line 334 may need adjustment with this defaulting to True.

Reply:
ack

Reply:
It might be helpful to also print something saying that we'll be using a prebuilt dart sdk, and if the developer wants to modify the Dart sources locally what they have to do.

Reply:
WDYT about `'Whether to use a prebuilt Dart SDK rather than compiling from source. This defaults to true and is enabled on CI.'`

Reply:
oh, it looks like I have to also add a `--no-prebuilt-dart-sdk` flag